antlr4/PostgreSQLParser.g4

```diff
@@ -24,6 +24,11 @@ parser grammar PostgreSQLParser;
 
 options { tokenVocab=PostgreSQLLexer; }
 
+
+root
+    : (stmt SEMI?)* EOF?
```

Comment:
A couple things:
  1) It should not be optional to end this with EOF. It needs to be required.
  2) File an issue to make `SEMI` required at the end of each `stmt` instead of using the `?` operator. This means having to alter the way the test files are generated, but that's easy. You don't have to do it, but the github issue makes things easy to remember.